zebrad/tests/acceptance.rs

```diff
@@ -53,12 +54,21 @@ fn generate_args() -> Result<()> {
     let output = child.wait_with_output()?;
     output.assert_failure()?;
 
+    // create config file path in the same location as binary
+    let binary_path = Path::new(env!("CARGO_BIN_EXE_zebrad"));
+    let path = format!(
+        "{}/{}",
+        binary_path.parent().unwrap().to_str().unwrap(),
+        "zebrad.toml"
+    );
+
     // Valid
-    let (child, _guard) = get_child(&["generate", "-o", "file.yaml"])?;
+    let (child, _guard) = get_child(&["generate", "-o", &path])?;
     let output = child.wait_with_output()?;
     output.assert_success()?;
 
-    // Todo: Check if the file was created
+    // Check if the file was created
+    assert_eq!(Path::new(&path).exists(), true);
```

Comment:
If you use `PathBuf`, you can call `exists` directly:
https://doc.rust-lang.org/std/path/struct.PathBuf.html#method.exists